### PR TITLE
Pypi fix, versioned SDK, improved README

### LINKED ISSUE #24: Installation problems v0.1.0
When I try to follow installation notes in [README](https://github.com/bblfsh/client-python/blob/master/README.md) I failed.

`pip3 install bblfsh` leads to the error:
```
pip3 install bblfsh
Collecting bblfsh
  Downloading bblfsh-0.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    make: *** No rule to make target 'deps'.  Stop.
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-m2t1aa3i/bblfsh/setup.py", line 16, in <module>
        subprocess.check_output(['make', 'deps'])
      File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
        **kwargs).stdout
      File "/usr/lib/python3.5/subprocess.py", line 708, in run
        output=stdout, stderr=stderr)
    subprocess.CalledProcessError: Command '['make', 'deps']' returned non-zero exit status 2

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-m2t1aa3i/bblfsh/
```

and 
```
git clone https://github.com/bblfsh/client-python.git
cd client-python
make install
```
leads to the next error:
```
Cloning into 'client-python'...
remote: Counting objects: 280, done.
remote: Compressing objects: 100% (17/17), done.
remote: Total 280 (delta 11), reused 21 (delta 8), pack-reused 255
Receiving objects: 100% (280/280), 131.22 KiB | 0 bytes/s, done.
Resolving deltas: 100% (132/132), done.
make: *** No rule to make target `install'.  Stop.
```

I was able to install it using
```
git clone https://github.com/bblfsh/client-python.git
cd client-python
pip3 install .
```

### LINKED ISSUE #25: Adding filter examples to the README
@juanjux thank you for the work on https://github.com/bblfsh/client-python/pull/23

Just opening an issue so that we add filter function (libuast) examples to the README because right now people will only discover it if they read the tests.

### PR DESCRIPTION
**Pull request by juanjux**
- Adds a `Manifest.in` file so pypi doesn't remove some files.
- Ensure the `libuast` directory is deleted before running `make deps`.
- Bump version (middle one, pypytest wasn't working so I could only test pypi bumping the last version).

Fixes #24.
Fixes #25.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
     sources.append('bblfsh/libuast/roles.c')
 
 # download c deoendencies
```
Comment (abeaumont) [inline setup.py]: unrelated, but s/deoendencies/dependencies/

File: bblfsh/pyuast.c [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#include "uast.h"
+#include "libuast/uast.h"
```
Comment (abeaumont) [inline bblfsh/pyuast.c]: I don't think this is correct, since `bblfsh` is not in the include list.

Comment (juanjux) [reply bblfsh/pyuast.c]: Yeah, cruft from one of my tests, works because gcc looks also form the source code file path, but unneeded.

**Comment by juanjux** [comment]
Changes from review feedback (thanks!) done, I've also included a dependencies section in the readme stating the need for libxml2 header files. Travis is failing for some reason, I'll investigating it.

**Pull request merged**